api/class-wc-rest-dev-product-variations-controller.php

```diff
@@ -28,4 +28,44 @@ class WC_REST_Dev_Product_Variations_Controller extends WC_REST_Product_Variatio
 	 */
 	protected $namespace = 'wc/v3';
 
+	/**
+	 * Get the image for a product variation.
+	 *
+	 * @param WC_Product_Variation $variation Variation
+	 * @return array
+	 */
+	protected function get_images( $variation ) {
+		if ( has_post_thumbnail( $variation->get_id() ) ) {
+			$attachment_id = $variation->get_image_id();
+		} else {
+			$attachment_id = current( $variation->get_gallery_image_ids() );
```

Comment:
By default product variations does not have galleries, this will be supported now on core?